jolt-core/src/lib.rs

```diff
@@ -9,11 +9,11 @@
 pub mod benches;
 pub mod lasso;
 mod msm;
-mod poly;
+pub mod poly;
```

Comment:
Not thrilled about all these new exports in order to support benchmarking from `cargo bench`.